kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/useFetch.js

```diff
@@ -0,0 +1,94 @@
+import get from 'lodash/get';
+import { ref, computed } from '@vue/composition-api';
+import { ViewMoreButtonStates } from '../../../../../constants';
+
+window.get = get;
+
+export default function useFetch(options) {
```

Comment:
Wondering if this should go in the coach/src/composables directory w/ it's cousin useFetchTree?

Reply:
Definetely, that seems like a good place!